javaCreateAccountCore/src/main/java/com/hamdan/forzenbook/java/createaccount/core/viewmodel/JavaCreateAccountViewModel.java

```diff
@@ -27,10 +30,15 @@ public class JavaCreateAccountViewModel extends ViewModel {
 
     private final BehaviorSubject<JavaCreateAccountState> state = BehaviorSubject.createDefault(new JavaCreateAccountState());
 
+    private final NavigatorJava navigator;
+
+    public Runnable onSuccess = null;
```

Comment:
You shouldn't need things like this. The view shouldn't be dictating the logic for you

Reply:
removed